changelogs/changelog.yaml

```
@@ -2,3 +2,11 @@ ancestor: null
 releases:
   0.0.0:
     release_date: '2022-01-29'
+  0.1.0:
+    changes:
+      release_summary: '| It''s a release!  First version to publish to Ansible Galaxy.
+
+        '
```

This could also be fixed by reverting and changing the `release_summary` to use `|-` in the YAML fragment and then re-running `antsibull-release`
```suggestion
      release_summary: It's a release! First version to publish to Ansible Galaxy.
```